src/spork.h

```diff
@@ -47,6 +47,8 @@ using namespace boost;
 #define SPORK_21_ENABLE_ZEROCOIN 10020
 #define SPORK_22_ZEROCOIN_MAINTENANCE_MODE 10021
 #define SPORK_23_STAKING_REQUIREMENTS 10022
+#define SPORK_24_QUERY_NODES 10023
```

Comment:
What is this spork used for?its not used anywhere in this pr afaik

Reply:
This one is for another upcoming feature Penple is looking into.

Reply:
Ok